doc/OnlineDocs/model_transformations/scaling.rst

```diff
@@ -0,0 +1,45 @@
+Model Scaling Transformation
+============================
+
+Good scaling of models can greatly improve the numerical properties of a problem and thus increase reliability and convergence. The ``ScaleModel`` transformation allows users to separate scaling of a model from the declaration of the model variables and constraints which allows for models to be written in more natural forms and to be scaled and rescaled as required without having to rewrite the model code.
```

Comment:
Instead of using the `ScaleModel` class name should we use the `core.scale_model` name provided to the `TransformationFactory`?